segmenters/nlp/JiebaSegmenter/__init__.py

```diff
@@ -14,20 +14,43 @@ class JiebaSegmenter(BaseSegmenter):
      - all
 
     :type mode: str
+    :param pool_size: Number of parallel processes to use.
+        The default is no parallel execution.
+        None will set this value to the number of CPUs.
+    :type pool_size: int
+    :param user_dict_file: Path to a custom dictionary.
+        This custom dictionary extends the default dictionary.
+        `See here for the format of the dict https://github.com/fxsjy/jieba#load-dictionary`_
+    :type user_dict_file: str
     :raises:
         ValueError: If `mode` is not any of the expected modes
+        FileNotFoundError: If `user_dict_file` does not exist
     :param args:  Additional positional arguments
     :param kwargs: Additional keyword arguments
 
     """
 
-    def __init__(self, mode: str = 'accurate', *args, **kwargs):
+    def __init__(self, mode: str = 'accurate', pool_size: int = 1, user_dict_file: str = None, *args, **kwargs):
         """Set Constructor."""
+        from multiprocessing import cpu_count
```

Comment:
Can you keep the imports in post_init? This way we can avoid some warnings when using jina